pages/en/guide/discovery.md

```diff
@@ -3,6 +3,14 @@ order: 10
 ---
 # Discovery
 
+During the discovery phase, you will connect your AWS account to the sunshower.io app using a secure vault with a password known only to you. These are the steps to make this happen:
```

Comment:
Sunshower.io should always be capitalized. The name of the platform is "the Sunshower platform"

Reply:
Fixed.